cli/src/index.ts

```diff
@@ -157,7 +157,7 @@ export function runProgram(config: Config): void {
     )
     .addOption(
       new Option(
-        '--signing-type <signingtype>',
+        '--signingtype <signingtype>',
```

Comment:
This is a breaking change - can we instead keep it as is and fix the variables?

Reply:
It's ok.
But, it is surely a breaking change same as using the `--signing-type` instead of `--signingtype` following the convention initially.